README.md

```diff
@@ -143,29 +194,42 @@ google_forms: [
 googleforms_googlesheet:
 ```
 
+- [ ] Make sure that `refresh-googleforms` is set to "yes".
+- [ ] Optionally, if you are saving data to Google, specify a googlesheet ID you'd like the GitHub data to be saved to. This will only be relevant if you've set `data_dest` to `google`.
+
 ### Setting up Slido
 
+This is assuming you have already exported Slido data to a google drive folder. Navigate to the Google Drive folder that has the Slidos you'd like to collect. Get the URL for this folder. It should look like this:
```

Comment:
Is there information from slido that can be linked to on exporting slido data to a google drive folder?

Reply:
Yes! Great idea!

Reply:
Done!